### DISCUSSION TITLE
iOS Voice Memos

### DISCUSSION THREAD

**Discussion by VadimNotJustDev** [notJust-dev/notjustdev#198] [Ideas]
![image](https://github.com/notJust-dev/DEVember/assets/16936043/334e3ade-2ddd-40d2-b232-f32392a33372)

**Comment by clintOnSky** [comment]
This would be awesome. I need to build an app for my school project that requires me to use audio playback and then storing the audio in local storage. I tried looking at the documentation for the Audio API and Expo FileSystem, and they were both kinda confusing
